pkgs/tools/compression/zdelta/default.nix

```diff
@@ -1,16 +1,24 @@
-{lib, stdenv, fetchurl}:
+{ lib, stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  name = "zdelta-2.1";
-  builder = ./builder.sh;
+  pname = "zdelta";
+  version = "2.1";
+
   src = fetchurl {
-    url = "${meta.homepage}/downloads/${name}.tar.gz";
+    url = "http://cis.poly.edu/zdelta/downloads/${pname}-${version}.tar.gz";
```

Comment:
Redirects to https://cse.engineering.nyu.edu/zdelta/downloads/zdelta-2.1.tar.gz and then 404s.

Reply:
Maybe you have automatic HTTPS redirection on. It curls fine.

Reply:
Using https://web.archive.org/web/20160316212948/http://cis.poly.edu/zdelta/ as the homepage URL.